netlify.toml

```diff
@@ -7,7 +7,7 @@ functions = "functions"
 command = "make non-production-build"
 
 [build.environment]
-HUGO_VERSION = "0.53"
+HUGO_VERSION = "0.57.2"
```

Comment:
@zacharysarah , Do you know how netlify is configured to use the extended version of Hugo? Just wondered.
Also, the other week I saw PR #16708 upgrade netlify for the `code` shortcode issue on the release-1.16 branch. I _think_ this PR upgraded Hugo to the latest version.

Has the Makefile been updated to this version of Hugo (for local testing)?

Reply:
@kbhawkey - Thanks for pointing that out! The makefile has not been updated, but from what I can see, the HUGO_VERSION in the makefile is only used when building the Docker container and when running  `scripts/hugo-version-check.sh`  to check that hugo versions are in sync between the makefile and netlify.toml.

Reply:
@kbhawkey 
> Do you know how netlify is configured to use the extended version of Hugo?

I don't. I'm not sure I know what "extended version" means in the context of Hugo--I'll do some research!

> I think this PR upgraded Hugo to the latest version.

Thanks for raising visibility on that PR, along with @aimeeu. That's a problematic PR for a host of unrelated reasons, and it's been reverted.

> Has the Makefile been updated to this version of Hugo (for local testing)?

Good catch, I'll do that now.

Reply:
I noticed that [Intermediate - view your changes locally](https://kubernetes.io/docs/contribute/intermediate/#view-your-changes-locally) states "Install Hugo version 0.53 or later."   That should change to something like "Install the Hugo version specified in website/netlify.toml."   Can you make that change in this PR or do you want a separate PR?

Reply:
@aimeeu To prevent scope creep, that’s a separate issue and PR. I’m happy to open those and link them to this PR.